README.md

```diff
@@ -115,51 +115,49 @@ You can specify all arguments above multiple times.
 ### Model Manipulation
 
 Sometimes, your model will output different results than what you expect.
-For example, `メロンパン` is split into two tokens in the following command.
+For example, `外国人参政権` is split into wrong tokens in the following command.
 We use `--scores` option to show the score of each character boundary:
 ```
-% echo '朝食はメロンパン1個だった' | cargo run --release -p predict -- --scores --model path/to/jp-0.4.7-5-tokenize.model.zst
-朝食 は メロン パン 1 個 だっ た
-0:朝食 -15398
-1:食は 24623
-2:はメ 30261
-3:メロ -26885
-4:ロン -38896
-5:ンパ 8162
-6:パン -23416
-7:ン１ 23513
-8:１個 18435
-9:個だ 24964
-10:だっ -15065
-11:った 14178
+% echo '外国人参政権と政権交代' | cargo run --release -p predict -- --scores --model path/to/jp-0.4.7-5-tokenize.model.zst
+外国 人 参 政権 と 政権 交代
+0:外国 -11785
+1:国人 16634
+2:人参 5450
+3:参政 4480
+4:政権 -3697
+5:権と 17702
+6:と政 18699
+7:政権 -12742
+8:権交 14578
+9:交代 -7658
 ```
 
-To concatenate `メロンパン` into a single token, manipulate the model in the following steps so that the score of `ンパ` becomes negative:
+The correct is `外国 人 参政 権`.
+To split `外国人参政権` into correct tokens, manipulate the model in the following steps so that the score of `参政権` becomes inverted:
 
 1. Dump a dictionary by the following command:
    ```
-   % cargo run --release -p manipulate_model -- --model-in path/to/jp-0.4.7-5-tokenize.model.zst --dump-dict path/to/dictionary.csv
+   % cargo run --release -p manipulate_model -- --model-in path/to/bccwj-suw+unidic.model.zst --dump-dict path/to/dictionary.csv
    ```
 
 2. Edit the dictionary.
 
-   The dictionary is a csv file. Each row contains a word, corresponding weights, and a comment in the following order:
+   The dictionary is a csv file. Each row contains a string pattern, corresponding weight array, and a comment in the following order:
 
-   * `right_weight` - A weight that is added when the word is found to the right of the boundary.
-   * `inside_weight` - A weight that is added when the word is overlapped on the boundary.
-   * `left_weight` - A weight that is added when the word is found to the left of the boundary.
+   * `word` - A string pattern (usually, a word)
+   * `weights` - A weight array. When the string pattern is contained in the input string, these weights are added to character boundaries of the range of the found pattern.
    * `comment` - A comment that does not affect the behaviour.
 
    Vaporetto splits a text when the total weight of the boundary is a positive number, so we add a new entry as follows:
    ```diff
-    メロレオストーシス,6944,-2553,5319,
-    メロン,8924,-10861,7081,
-   +メロンパン,0,-100000,0,melon🍈 bread🍞 in English.
-    メロン果実,4168,-1165,3558,
-    メロヴィング,6999,-15413,7583,
+    参撾,3167 -6074 3790,
+    参政,3167 -6074 3790,
+   +参政権,10000 -10000 10000 10000,参政/権
```

Comment:
I think the first and last weights of 10000 are confusing.
I suggest to set these to 0 and explain that weight 0 has no effect.